SpringKonstant/src/org/usfirst/frc6357/robotcode/subsystems/TFMini.java

```diff
@@ -44,7 +44,7 @@
  * @author hoo761
  */
 
```

Comment:
By changing the class to extend thread, we override the run method and remove our old start method. Now, calling .start() on a TFMini object starts a thread.